cvts/_data_retrieval.py

```diff
@@ -31,19 +29,22 @@ def vehicle_trace(vehicle_id:int, from_instant:int, to_instant:int)->dict:
                 'type':    'via'
             }
     """
-    dates = [datetime.fromtimestamp(epochtime) for epochtime in range(from_instant, to_instant + DAY, DAY)]
     data = []
     for date in dates:
-        fldr = join(DATA_LAKE, str(date.year), str(date.month).zfill(2), datetime.strftime(date, "%Y%m%d"))
+        fldr = join(
+            RAW_PATH,
+            str(date.month).zfill(2),
+            date.strftime("%Y%m%d"))
         if not isdir(fldr):
             continue
-        file = join(fldr, f'vehicle_{vehicle_id}.gzip')
-        if not isfile(file):
+        filename = join(fldr, '{}.gzip'.format(vehicle_id))
+        if not isfile(filename):
             continue
-        data.append(pd.read_csv(file))
-    df = pd.concat(data)
-    df = df.loc[(df.datetime>=from_instant)&(df.datetime<=to_instant), :]
```

Comment:
With this gone, we are losing a feature that is to only return data for entire days. Is that what we want?

Reply:
Yes.